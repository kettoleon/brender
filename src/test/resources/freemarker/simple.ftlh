So, you are living at Town: ${communicationBody.address.town}, Street: ${communicationBody.address.street}