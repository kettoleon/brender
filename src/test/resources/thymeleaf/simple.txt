So, you are living at [# th:object="${communicationBody.address}"]Town: [(*{town})], Street: [(*{street})][/]