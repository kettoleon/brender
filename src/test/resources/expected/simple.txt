So, you are living at Town: townName, Street: streetName